packages/squiggle-lang/src/ast/peggyParser.peggy

```diff
@@ -24,7 +24,7 @@ outerBlock
 importStatementsList = (@importStatement __nl)*
 
 importStatement
-  = _nl 'import' __ file:string variable:(__ 'as' __ @identifier)
+  = _nl 'import' __nl file:string variable:(__nl 'as' __nl @identifier)
```

Comment:
Previously, this wasn't allowed, now it's allowed:
```
import
"foo"
as
foo
```

Reply:
Is there any reason why we might actively want this to be allowed, or is it more the case that we're neutral on it or similar?

Reply:
Mostly neutral. It might matter when the import string is long and so prettier would prefer to break it, or something.

I'm just documenting minor consequences that are not 100% backward/forward-compatible.